ivy_tests/test_ivy/test_frontends/test_jax/test_jax_devicearray.py

```diff
@@ -613,3 +613,541 @@ def test_jax_special_rrshift(
+    ret_gt = helpers.flatten_and_to_np(ret=ret_gt)
+    for (u, v) in zip(ret, ret_gt):
+        helpers.value_test(
+            ret=ret,
+            ret_from_gt=ret_gt,
+            ground_truth_backend="jax",
+        )
+
+
+# __rtruediv__
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric", full=True),
+        shared_dtype=True,
+        num_arrays=2,
+    )
+)
+def test_jax_special_rtruediv(
+    dtype_x,
+    fw,
+):
+    input_dtype, x = dtype_x
+    data = DeviceArray(x[0])
+    other = DeviceArray(x[1])
+    ret = data.__rtruediv__(other)
+    ret_gt = jnp.array(x[1], dtype=input_dtype[1]) / jnp.array(
+        x[0], dtype=input_dtype[0]
+    )
+    ret = helpers.flatten_and_to_np(ret=ret)
+    ret_gt = helpers.flatten_and_to_np(ret=ret_gt)
+    for (u, v) in zip(ret, ret_gt):
+        helpers.value_test(
+            ret=ret,
+            ret_from_gt=ret_gt,
+            ground_truth_backend="jax",
+        )
+
+
+# __mod__
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float", full=True),
+        shared_dtype=True,
+        num_arrays=2,
+    )
+)
+def test_jax_special_mod(
+    dtype_x,
+    fw,
+):
+    input_dtype, x = dtype_x
+    data = DeviceArray(x[0])
+    other = DeviceArray(x[1])
+    ret = data.__mod__(other)
+    ret_gt = jnp.mod(
+        jnp.array(x[0], dtype=input_dtype[0]), jnp.array(x[1], dtype=input_dtype[1])
+    )
+    ret = helpers.flatten_and_to_np(ret=ret)
+    ret_gt = helpers.flatten_and_to_np(ret=ret_gt)
+    for (u, v) in zip(ret, ret_gt):
+        helpers.value_test(
+            ret=ret,
+            ret_from_gt=ret_gt,
+            ground_truth_backend="jax",
+        )
+
+
+# __rmod__
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float", full=True),
+        shared_dtype=True,
+        num_arrays=2,
+    )
+)
+def test_jax_special_rmod(
+    dtype_x,
+    fw,
+):
+    input_dtype, x = dtype_x
+    data = DeviceArray(x[0])
+    other = DeviceArray(x[1])
+    ret = data.__rmod__(other)
+    ret_gt = jnp.remainder(
+        jnp.array(x[1], dtype=input_dtype[1]), jnp.array(x[0], dtype=input_dtype[0])
+    )
+    ret = helpers.flatten_and_to_np(ret=ret)
+    ret_gt = helpers.flatten_and_to_np(ret=ret_gt)
+    for (u, v) in zip(ret, ret_gt):
+        helpers.value_test(
+            ret=ret,
+            ret_from_gt=ret_gt,
+            ground_truth_backend="jax",
+        )
+
+
+# __divmod__
+@handle_cmd_line_args
+@given(
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float", full=True),
+        shared_dtype=True,
+        num_arrays=2,
+    )
+)
+def test_jax_special_divmod(
+    dtype_x,
+    fw,
+):
+    input_dtype, x = dtype_x
+    data = DeviceArray(x[0])
+    other = DeviceArray(x[1])
+    ret = data.__divmod__(other)
```

Comment:
this is a python built-in function (like `abs()`), would you mind trying to test it with `divmod(x,y)` to verify that the function correctly links to the special method? (just like how arithmetic operator will work)